src/Client.cpp

```diff
@@ -27,9 +27,9 @@
 #include <fastdds/dds/subscriber/DataReader.hpp>
 #include <fastdds/dds/subscriber/qos/DataReaderQos.hpp>
 #include <fastdds/dds/subscriber/SampleInfo.hpp>
-#if FASTRTPS_VERSION_MINOR >= 2
+#if FASTRTPS_VERSION_MAJOR >= 2 && FASTRTPS_VERSION_MINOR >= 2

```

Comment:
It is not necessary to check the major version as it will not compile with any version lower than 2.0.0 due to the DDS API

Reply:
Same for the Server and Subscriber

Reply:
I think it is clearer (eve when useless) to have it this way.